src/components/Button/style.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
```

Comment:
Acredito que aqui poderia ser um `button` ao inves de uma div.

```suggestion
export const Wrapper = styled.button`
```